sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/ApplicationDetailsForCOE.vue

```diff
@@ -90,20 +102,55 @@ export default {
     const menu = ref();
     const items = ref([] as MenuType[]);
     const showModal = ref(false);
+    const formioUtils = useFormioUtils();
     const editCOEModal = ref({} as ModalDialog<boolean>);
     const denyCOEModal = ref({} as ModalDialog<void>);
-    const showHideConfirmCOE = () => {
-      showModal.value = !showModal.value;
-    };
-
+    const confirmCOEModal = ref({} as ModalDialog<FormIOForm | boolean>);
+    const TUITION_REMITTANCE_AMOUNT = "tuitionRemittanceAmount";
     const loadInitialData = async () => {
       initialData.value =
         await ConfirmationOfEnrollmentService.shared.getApplicationForCOE(
           props.disbursementScheduleId,
           props.locationId,
         );
     };
-
+    const showHideConfirmCOE = async () => {
+      const modalResult = await confirmCOEModal.value.showModal();
+      if (!modalResult) {
+        return;
+      }
+      try {
+        const tuitionRemittanceAmount = formioUtils.getComponentValueByKey(
```

Comment:
The `modalResult` has the form.io submitted data, please get the value from there and remove the `TUITION_REMITTANCE_AMOUNT`.